src/components/Fabric/Fabric.tsx

```diff
@@ -49,6 +44,11 @@ export class Fabric extends React.Component<React.HTMLProps<HTMLDivElement>, IFa
   public componentDidMount() {
     this._events.on(document.body, 'mousedown', this._onMouseDown, true);
     this._events.on(document.body, 'keydown', this._onKeyDown, true);
+
+    // Ensure that the HTML element has a dir specified. This helps to ensure RTL/LTR macros in css for all components will work.
+    if (document && document.documentElement && !document.documentElement.getAttribute('dir')) {
```

Comment:
Anna; I think this might be a bad idea. Think of it like this:

Control will render. It will go to the DOM and then to pixels on the screen, without DIR attribute set. So you will potentially see screwed up state for a moment. Then the component will mount, and it will fix the issue causing another layout and visual glitch.

Instead... what if you just made this more server safe? Example:

```js
if (typeof(document) === 'object' && ...)
```

and then keep it where it is in global scope? That way it doesn't mess with anything in server or tests and stays safe for the scenarios where it's needed and executes before rendering.